test/e2e/config/docker.yaml

```diff
@@ -110,11 +110,11 @@ providers:
 variables:
   # default variables for the e2e test; those values could be overridden via env variables, thus
   # allowing the same e2e config file to be re-used in different prow jobs e.g. each one with a K8s version permutation
-  KUBERNETES_VERSION: "v1.19.11"
+  KUBERNETES_VERSION: "v1.21.2"
   ETCD_VERSION_UPGRADE_TO: "3.4.9-0"
   COREDNS_VERSION_UPGRADE_TO: "1.7.0"
```

Comment:
Should these be updated too?

Reply:
Based on the kubeadm defaults I would use etcd 3.4.14-0 and CoreDNS 1.8.0:
ref: https://github.com/pacoxu/kubernetes/blob/master/cmd/kubeadm/app/constants/constants.go#L467-L469

Reply:
Ah nope. I should probably choose etcd v3.4.16 and CoreDNS 1.8.4 so we actually have an update?

Reply:
As no image newer then etcd v3.4.13 has been pushed, I'll downgrade to v3.4.13 again